entrypoint/dev.js

```diff
@@ -33,7 +33,8 @@ const wasi = new WASI({
   },
 });
 
-const socket = new ReconnectingWebSocket("ws://127.0.0.1:8080/watcher");
+const port = document.currentScript.getAttribute("data-port") || 8080;
+const socket = new ReconnectingWebSocket(`ws://127.0.0.1:${port}/watcher`);
```

Comment:
This should make passing the port unnecessary.
```suggestion
const socket = new ReconnectingWebSocket(`ws://${location.host}/watcher`);
```

Reply:
I concur, let's use `location.host`, which means that passing the port to `indexPage` and `data-port` attribute won't be needed.